packages/core/src/api/exporters/copyExtension.ts

```diff
@@ -17,46 +52,79 @@ export const createCopyToClipboardExtension = <
   Extension.create<{ editor: BlockNoteEditor<BSchema, I, S> }, undefined>({
     name: "copyToClipboard",
     addProseMirrorPlugins() {
-      const tiptap = this.editor;
-      const schema = this.editor.schema;
       return [
         new Plugin({
           props: {
             handleDOMEvents: {
-              copy(_view, event) {
+              copy(view, event) {
                 // Stops the default browser copy behaviour.
                 event.preventDefault();
                 event.clipboardData!.clearData();
 
-                const selectedFragment =
-                  tiptap.state.selection.content().content;
-
-                const internalHTMLSerializer = createInternalHTMLSerializer(
-                  schema,
-                  editor
-                );
-                const internalHTML =
-                  internalHTMLSerializer.serializeProseMirrorFragment(
-                    selectedFragment
-                  );
-
-                const externalHTMLExporter = createExternalHTMLExporter(
-                  schema,
-                  editor
-                );
-                const externalHTML =
-                  externalHTMLExporter.exportProseMirrorFragment(
-                    selectedFragment
+                // Checks if a `blockContent` node is being copied and expands
```

Comment:
As there are no tests specific for this, can you add a comment why we need this? Why did we make this change, i.e.: what scenario would break without expanding the selection in this way?